runtime/src/bank/bank_hash_details.rs

```diff
@@ -35,7 +77,7 @@ pub(crate) struct BankHashDetails {
     pub accounts: BankHashAccounts,
 }
 
-impl BankHashDetails {
+impl BankHashSlotDetails {
```

Comment:
It contains the accounts too, so "bankhash" does not really describe it. How about `BankSlotDetails` (and `BankHashDetails` becomes `BankDetails`?